lib/suspenders/app_builder.rb

```diff
@@ -330,8 +330,9 @@ def configure_puma
     end
 
     def set_up_forego
-      copy_file 'sample.env', '.sample.env'
-      copy_file 'Procfile', 'Procfile'
+      copy_file "sample.env", ".sample.env"
```

Comment:
It looks like we don't need this line. Want to remove and squash your commits together?